llama.h

```diff
@@ -81,7 +81,7 @@ extern "C" {
     typedef void (*llama_progress_callback)(float progress, void *ctx);
 
    struct llama_context_params {
-        int seed;                              // RNG seed, -1 for random
+        unsigned int seed;                     // RNG seed, -1 for random
         int n_ctx;                             // text context
         int n_batch;                           // prompt processing batch size
         int n_gpu_layers;                      // number of layers to store in VRAM
```

Comment:
Avoid `unsigned int`. Use `uint32_t` and `int32_t`

```suggestion
        uint32_t seed;                             // RNG seed, -1 for random
        int32_t  n_ctx;                            // text context
        int32_t  n_batch;                          // prompt processing batch size
        int32_t  n_gpu_layers;                     // number of layers to store in VRAM
```